### PR TITLE
Pos addons 10.0 pos printer network

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

File: pos_restaurant_base/doc/index.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
+==========================
+ Hardware Network Printer
+==========================
+
+Technical module in POS.
+The standard Printer class does not allow you to override the functions of this class.
+This module copies the Printer class and sets the default printers and allows you to override the Printer class.
```
Comment (yelizariev) [inline pos_restaurant_base/doc/index.rst]: What means ``sets the default printers``?

File: pos_printer_network/doc/index.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  * Click ``[Save]``
+
+* go to ``Point of Sale >> Point of Sale``
+
+  * Open the POS in Configuration
+  * Click on ``[Edit]``
+  * In Order Printers Add an item (New printer)
+  * Click ``[Save]``
+  * Specify the proxy server IP address of the POS Box
+  * Specify the proxy server IP address of the Network printer for receipt printer
```
Comment (yelizariev) [inline pos_printer_network/doc/index.rst]: IP address of Network printer is nto a ``proxy server IP``

File: hw_printer_network/README.rst [hunk 3]
```diff
@@ -0,0 +1,29 @@
+==========================
+ Hardware Network Printer
+==========================
+
+Hardware Network Printer
```
Comment (yelizariev) [inline hw_printer_network/README.rst]: {Put some short introduction first.}

{Then add more detailed description, technical specifications, any other information that could be interested for other developers. Don't forget that Usage instructions is a separated and has to be located in doc/index.rst file.}

File: pos_printer_network/README.rst [hunk 4]
```diff
@@ -0,0 +1,29 @@
+=====================
+ POS Network Printer
+=====================
+
+POS Network Printer
```
Comment (yelizariev) [inline pos_printer_network/README.rst]: {Put some short introduction first.}

{Then add more detailed description, technical specifications, any other information that could be interested for other developers. Don't forget that Usage instructions is a separated and has to be located in doc/index.rst file.}

File: pos_restaurant_base/doc/index.rst (hunk 1 above)
Comment (GabbasovDinar) [inline pos_restaurant_base/doc/index.rst]: @yelizariev я имел ввиду, что вместо старого класса Printer будет использоваться новый

Comment (yelizariev) [inline pos_restaurant_base/doc/index.rst]: Clarify description

**Pull request merged**